ui/app/templates/components/console/ui-panel.hbs

```diff
@@ -1,6 +1,8 @@
-<button type="button" class="button is-ghost console-close-button" {{action "closeConsole"}} data-test-console-panel-close>
-  <Icon @name="x" aria-label="Close console" />
-</button>
+<div class="console-close-button">
+  <button type="button" class="button is-ghost" {{action "closeConsole"}} data-test-console-panel-close>
```

Comment:
If you wanted to add a helper "has-right-margin-xs". There are two we use now "has-right-margin-m" and "has-right-margin-l". Totally up to you.

Reply:
Sounds good. I use those quite often but I typically will use helpers to avoid having to create a class. Do you think there is an advantage to removing the margin in favor of a helper here?

Reply:
I suppose in the case of the margin it could be removed since it's the only rule directly on the button but removing the flex rules from `console-close-button` doesn't allow us to remove the class. Have we established a guideline on how much we should lean on helper classes as opposed to defined classes for an element and whether or not we should be mixing them?

Reply:
It's a great question. I'll add it to UI sync.